src/mmw/js/src/data_catalog/templates/resultDetails.html

```diff
@@ -0,0 +1,20 @@
+<div class="result">
+    <div class="result-detail-header">
+        <h2>
+            {{ title }}
+        </h2>
+        <button type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+        </button>
+        <a href="#"><i class="fa fa-search-plus"></i> Zoom to extent</a>
+    </div>
+    <hr>
+    {% if author %}
+        <p>
+            <i class="fa fa-user"></i>{{ author }}
+        </p>
+    {% endif %}
+    <p>
+        {{ description }}
```

Comment:
We could also check for `begin_date` and `end_date` if they're not present, fall back to to `created_at`. That way we wouldn't have to worry about which catalog is active.